test/stripe/charge_refund_test.rb

```diff
@@ -0,0 +1,47 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class ChargeRefundTest < Test::Unit::TestCase
+    should "refunds should be listable" do
+      @mock.expects(:get).once.returns(make_response(make_charge))
+
+      charge = Stripe::Charge.retrieve('test_charge')
+
+      assert charge.refunds.first.kind_of?(Stripe::Refund)
+    end
+
+    should "refunds should be refreshable" do
+      @mock.expects(:get).twice.returns(make_response(make_charge), make_response(make_refund(:id => 'refreshed_refund')))
+
+      charge = Stripe::Charge.retrieve('test_charge')
+      refund = charge.refunds.first
+      refund.refresh
+
+      assert_equal 'refreshed_refund', refund.id
+    end
+
+    should "refunds should be updateable" do
+      @mock.expects(:get).once.returns(make_response(make_charge))
+      @mock.expects(:post).once.returns(make_response(make_refund(:metadata => {'key' => 'value'})))
+
+      charge = Stripe::Charge.retrieve('test_charge')
+      refund = charge.refunds.first
+
+      assert_equal nil, refund.metadata['key']
+
+      refund.metadata['key'] = 'valu'
```

Comment:
`valu` should be `value`. But this test passes anyway :(. This assignment doesn't do anything useful, because we return what we specify in the `expects` line above regardless. Can you add a `with` clause there?